README.md

```diff
@@ -1 +1,254 @@
-# Microverse-Portfolio-Project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse Portfolio Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#hello microverse)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#html-css)
+    - [Key Features](#home-page)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#amen-musingarimi)
+- [🔭 Future Features](#the other pages)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#microverse)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Microverse-Portfolio-Project] <a name="Portfolio Website"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[Microverse-Portfolio-Project]** is a portifolio website
+
+## 🛠 Built With <a name="html and css"></a>
+
+### Tech Stack <a name="html and css"></a>
+
+> This is my first Microverse where I basically test my knowledge of google workflows and the linters.
+
+<details>
+  <summary>A test project</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
```

Comment:
- Please replace this with the right Tech stack and delete the other ones that aren't relevant to this project :+1: